core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -2261,7 +2261,7 @@ class KafkaController(val config: KafkaConfig,
     val brokerEpochOpt = controllerContext.liveBrokerIdAndEpochs.get(brokerId)
     if (brokerEpochOpt.isEmpty) {
       info(s"Ignoring AlterIsr due to unknown broker $brokerId")
-      callback.apply(Right(Errors.STALE_BROKER_EPOCH))
+      callback.apply(Right(Errors.UNKNOWN_MEMBER_ID))
```

Comment:
not sure whether ```UNKNOWN_MEMBER_ID`` is more suitable than ```STALE_BROKER_EPOCH```.

Reply:
Thank you for your reply,  here we have 2 judgement. the first is `if (brokerEpochOpt.isEmpty)` and second is `if(!brokerEpochOpt.contains(brokerEpoch))`, they both return `STALE_BROKER_EPOCH `. maybe the first is meant to return`UNKNOWN_MEMBER_ID `.

Reply:
You are right, I just inspect The KIP-497 and there isn't a `UNKNOWN_MEMBER_ID`.